userguide/dailyuse/anbox.rst

```diff
@@ -5,18 +5,19 @@ Run android applications
 
 .. note::
     - Anbox is in early development
-    - Anbox for Ubuntu Touch is in even more early development
     - Anbox only works on the 16.04 version of Ubuntu Touch, which is also in early development
-    - Only selected devices are supported for the moment, more will be added in the future.
 
 Supported devices
 -----------------
 
 Make sure your device is supported:
 
-Meizu Pro 5 (codename: *turbo*, name of the boot partition: *bootimg*)
-BQ M10 HD (codename: *cooler*, name of the boot partition: *boot*)
-BQ M10 FHD (codename: *frieza*, name of the boot partition: *boot*)
+- Meizu Pro 5 (codename: *turbo*, name of the boot partition: *bootimg*)
+- Fairphone 2 (codename: "FP2", name of the boot partition: *boot*)
+- Oneplus one (codename: "bacon", name of the boot partition: *boot*)
+- Nexus 5 (codename: "hammerhead", name of the boot partition: *boot*)
+- BQ M10 HD (codename: *cooler*, name of the boot partition: *boot*)
+- BQ M10 FHD (codename: *frieza*, name of the boot partition: *boot*)
```

Comment:
Please use the same style of marking all devices here. Some have `""` and others have `**`. I think the document would read better if you replaced all of them with two backticks on each side (``)